.travis.yml

```diff
@@ -6,11 +6,10 @@ install:
   - cp config/database.yml.example config/database.yml
   - cp config/config.yml.example config/config.yml
   - cp db/schema.rb.example db/schema.rb
-  - docker-compose build
-  - docker-compose run web sleep 10
```

Comment:
Using exec instead of run, since run commands creates a new container everytime.